docker_daemon/check.py

```diff
@@ -589,18 +586,17 @@ def _report_container_count(self, containers_by_id):
 
         for state in per_state_count:
             if state:
-                m_func(self, 'docker.container.count', per_state_count[state], tags=['container_state:%s' % state.lower()])
+                self.gauge('docker.container.count', per_state_count[state], tags=['container_state:%s' % state.lower()])
 
     def _report_volume_count(self):
         """Report volume count per state (dangling or not)"""
-        m_func = FUNC_MAP[GAUGE][self.use_histogram]
 
         attached_volumes = self.docker_client.volumes(filters={'dangling': False})
         dangling_volumes = self.docker_client.volumes(filters={'dangling': True})
         attached_count = len(attached_volumes.get('Volumes', []) or [])
         dangling_count = len(dangling_volumes.get('Volumes', []) or [])
-        m_func(self, 'docker.volume.count', attached_count, tags=['volume_state:attached'])
-        m_func(self, 'docker.volume.count', dangling_count, tags=['volume_state:dangling'])
+        self.gauge('docker.volume.count', attached_count, tags=['volume_state:attached'])
+        self.gauge('docker.volume.count', dangling_count, tags=['volume_state:dangling'])
```

Comment:
ditto, maybe this one makes sense per container? Maybe not but we should give it some thought.